src/SourceBuild/content/repo-projects/Directory.Build.targets

```
@@ -335,6 +335,33 @@
              RunEachTargetSeparately="$(RunEachTargetSeparately)" />
   </Target>
 
+  <!-- Discover the Arcade SDKs built from the live Arcade source and set them as overrides here. -->
+  <Target Name="DiscoverArcadeSourceBuiltSdkOverrides"
+          DependsOnTargets="GetRepositoryReferenceInfo"
```

This target also depends on the dependent repositories being built which is a parent of `GetRepositoryReferenceInfo`.

```suggestion
          DependsOnTargets="BuildRepoReferences"
```